man/repart.d.xml

```diff
@@ -422,6 +422,22 @@
         <xi:include href="version-info.xml" xpointer="v247"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MkfsOptions=</varname></term>
+
+        <listitem><para>Takes a list of command line options to be passed to <literal>mkfs</literal> at file
+        system creation time. Options are specific to the file system type specified by
```

Comment:
hmm, so this sounds as if we invoked the "mkfs" binary. We don't do that. We always invoke the various "mkfs.<fstype>" binaries directly. And that matters, because that means we put together a cmdline for those, which is different from the "mkfs" command.

This needs to be made clear.

Also use `<command></command>` for commands

Reply:
Doc moved to different PR